src/ray/stats/stats.h

```diff
@@ -50,14 +50,23 @@ static inline void Init(
     RAY_LOG(INFO) << "Disabled stats.";
     return;
   }
+  RAY_LOG(INFO) << "Initialize stats";
```

Comment:
Just wondering how useful is this log: should we make it a DEBUG, or add more information if we think this is useful to log?